### PR TITLE
Fix neutron-gateway verifier

### PR DESCRIPTION
**Pull request by r-f-g**
- replace `get-status-routers` w/ `show-routers`
 - replace `get-status-dhcp` w/ `show-dhcp-networks`
 - replace `get-status-lb` w/ `show-loadbalancers`
 - using yaml to load actions output instead of json
 - change custom charm to openstack-charmers-next

fixes: [#1944510](https://bugs.launchpad.net/juju-verify/+bug/1944510)

### CODE REVIEW

**Comment by r-f-g** [comment]
I deleted it in accident.

**Review by r-f-g — COMMENTED** [review]
recheck-full

**Review by r-f-g — COMMENTED** [review]
recheck-full

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Juju-verify requires Juju 2.8.10 or higher.
 * nova-compute (Usable with the next stable release of the charm. Currently available as a [nova-compute-rc])
 * ceph-osd (Usable with the next stable release of the charm. [cs:~openstack-charmers-next/ceph-osd] and [cs:~openstack-charmers-next/ceph-mon])
 * ceph-mon (Usable with the next stable release of the charm. [cs:~openstack-charmers-next/ceph-mon])
-* neutron-gateway (Usable with the custom release of the charm in martin-kalcok namespace. [cs:~/martin-kalcok/neutron-gateway-2])
+* neutron-gateway (Usable with the custom release of the charm in martin-kalcok namespace. [cs:~openstack-charmers-next/neutron-gateway])
```
Comment (mkalcok) [inline README.md:16]: Remove text reference to my namespace

**Review by mkalcok — CHANGES_REQUESTED** [review]
reference to `neutron-gateway` charm needs to be updated also in the `docs/getting-started.rst`

**Review by mkalcok — APPROVED** [review]
LGTM

**Pull request merged**